package.json

```diff
@@ -18,7 +18,7 @@
     "url": "https://github.com/prismatic-io/translations.git"
   },
   "license": "MIT",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
Should this be a patch version update? We're not necessarily adding new things just fixing behavior.